src/cpp/server/backends/kokoro_server.cpp

```
@@ -21,17 +21,39 @@ using namespace lemon::utils;
 namespace lemon {
 namespace backends {
 
-InstallParams KokoroServer::get_install_params(const std::string& /*backend*/, const std::string& version) {
+namespace {
+// Kokoro doesn't expose backend selection through RuntimeConfig; we resolve it
+// from the host platform so each OS gets the right binary archive.
+std::string default_kokoro_backend() {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)